index.html

```diff
@@ -0,0 +1,1729 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <link
+      href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://use.fontawesome.com/releases/v5.13.0/js/all.js"
+      data-auto-replace-svg="nest"
+    ></script>
+    <title>NewsWeek</title>
+  </head>
+  <body>
+    <header class="sticky-top">
+      <div class="container-fluid nav-container">
+        <div class="row pt-2 pb-2">
+          <div class="col weather d-none d-lg-block mt-1">
+            <a href="#"> <img src="images/weather_date.png" alt="" /></a>
+            <p>Mon, Jun 08, 2020</p>
+          </div>
+          <div class="col left-nav align-self-center">
+            <a href="#">
+              <img
+                class="d-lg-none"
+                src="images/newsweek_logo.png"
+                alt="newsweek_logo"
+            /></a>
+            <div class="wrapper big-logo">
+              <a href="#">
+                <img
+                  class="d-none d-lg-block"
+                  src="images/newsweek_logo.png"
+                  alt="newsweek_logo"
+              /></a>
+            </div>
+          </div>
+          <div
+            class="col d-flex right-nav align-items-center justify-content-end"
+          >
+            <i class="fa fa-user mx-1 d-md-none"></i>
+            <h5 class="d-none d-md-block nav-login mt-2">
+              <a href="#"> LOGIN</a>
+            </h5>
+            <button type="submit" class="btn btn-warning mx-3 py-1 px-2">
+              SUBSCRIBE >
+            </button>
+            <!-- <img class="mx-1 mt-1" src="images/subscribe.png" alt="subscribe" /> -->
```

Comment:
- Remove commented code. It’s better to delete commented code. If you need to review a previous version of your code, use git history (screenshot from Pathwright)
  ![commented code](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)

Reply:
Hello Ghosh, can you open slack and check my message please? Thank you!